WEEKLY_MEETINGS.md

```diff
@@ -0,0 +1,10 @@
+We've agreed that we'll have a weekly meeting, every Thursday at 09:00. Not all the members have to be present. If we expect everybody to be there, scheduling can get really challenging, and this won't get easier with time, it will only get more difficult as we grow. Also, is important to keep doing it continiously so into becomes part of our routine.
```

Comment:
Also, it is important to keep doing it continuously so it becomes part of our routine.